test/Microsoft.SqlTools.ServiceLayer.UnitTests/DisasterRecovery/BackupUtilitiesStub.cs

```diff
@@ -0,0 +1,61 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.Admin;
+using Microsoft.SqlTools.ServiceLayer.DisasterRecovery;
+using Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts;
+using System.Data.SqlClient;
+using System.Threading;
+
+namespace Microsoft.SqlTools.ServiceLayer.UnitTests.DisasterRecovery
+{
+    /// <summary>
+    /// Stub class that implements IBackupUtilities
+    /// </summary>
+    public class BackupUtilitiesStub : IBackupUtilities
+    {
+        /// <summary>
+        /// Initialize 
+        /// </summary>
+        /// <param name="dataContainer"></param>
+        /// <param name="sqlConnection"></param>
+        public void Initialize(CDataContainer dataContainer, SqlConnection sqlConnection)
+        {
+        }
+
+        /// <summary>
+        /// Return database metadata for backup
+        /// </summary>
+        /// <param name="databaseName"></param>
+        /// <returns></returns>
+        public BackupConfigInfo GetBackupConfigInfo(string databaseName)
+        {
+            return null;
+        }
+
+        /// <summary>
+        /// Set backup input properties
+        /// </summary>
+        /// <param name="input"></param>
+        public void SetBackupInput(BackupInfo input)
+        {
+        }
+
+        /// <summary>
+        /// Execute backup
+        /// </summary>
+        public void PerformBackup()
+        {
+            Thread.Sleep(2000);
```

Comment:
>2000 [](start = 25, length = 4)

you might see if a shorter delay is sufficient for this test case.  Perhaps 200ms or 500ms, just so the tests run a little faster.